op-conductor/conductor/service.go

```diff
@@ -609,6 +656,24 @@ func (oc *OpConductor) queueAction() {
 func (oc *OpConductor) handleLeaderUpdate(leader bool) {
 	oc.log.Info("Leadership status changed", "server", oc.cons.ServerID(), "leader", leader)
 
+	// Update rollup boost execution mode based on leadership status
+	if oc.rollupBoost != nil {
+		mode := false
+		if leader {
+			mode = true
+		}
+
+		// Use a background context since this is an async operation
+		ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+		defer cancel()
+
+		if err := oc.rollupBoost.SetExecutionMode(ctx, mode); err != nil {
```

Comment:
same, what happens if this fails, how do we make sure that we can transferLeader when this failed?